### DISCUSSION TITLE
Im getting only java and minecraft suggestions in the "For you" feed. I have nothing todo with any of both

### DISCUSSION THREAD

**Discussion by OfficialLambdax** [community/community#20427] [Other Feature Feedback, Questions, & Ideas] [answered]
Github be like:
The user is publishing autoit code that has something todo with tcp and something with encryption. Hmmm... Lets feed him minecraft <3

For real: Ive never published even the slightest java code or star'd any of that, nor am i looking into these things because that is not my language. i just once published a bug report for a mod of that game. But yet EVERY post in the feed is filled with minecraft and java.

https://www.youtube.com/watch?v=rYQxiGZ14h8

**Comment by FahdSeddik** [comment, marked as answer]
Hey!
I am pretty sure it is based more on who you are following. Hope this helps!

**Reply by OfficialLambdax** [reply]
I am not following anyone and never had yet. But maybe if i do so then github decides to feed me something more fitting. Thanks for your answer o/

**Reply by FahdSeddik** [reply]
No problem! 
Also, can you mark it as an answer so people can see it more easily. Thanks

**Comment by jfullstackdev** [comment]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
